### PR TITLE
:sparkles: feat: footer added

### PR DESCRIPTION
**Pull request by atilasousa**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206109714345741

### CODE REVIEW

File: components/layout/footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    padding="4rem 0.75rem 1.5rem 0.75rem"
+  >
+    <Box
+      textAlign="center"
+      mb="l"
+      maxWidth="96%"
+      minWidth="90rem"
+      fontSize="inherit"
+      fontFamily="inherit"
+      justifyContent="center"
```
Comment (MarioBatalha) [inline components/layout/footer/index.tsx]: This justifyContent doesn't have any effect cuz you're not using display.
U can remove it.

File: components/layout/footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        variant={'display'}
+        fontFamily="inherit"
+        justifyContent="center"
+      >
+        built with
+        <HeartSVG maxHeight="1.125rem" maxWidth="1.125rem" width="100%" />
+        By Interest Protocol
+      </Typography>
+    </Box>
+    <Box textAlign="center" my="2xl" fontSize="inherit" fontFamily="inherit">
```
Comment (MarioBatalha) [inline components/layout/footer/index.tsx]: This textAlign doesn't make sense cuz inside of component you have tag like div, instead use display.

File: components/layout/footer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+      <Box fontFamily="Proto">Follow us</Box>
```
Comment (MarioBatalha) [inline components/layout/footer/index.tsx]: Use Typography and add textAlign="center"

File: components/layout/footer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+            p="xs"
+            width="2.5rem"
+            height="2.5rem"
+            border="1px solid"
+            borderRadius="full"
+            borderColor="outline"
+          >
+            <DiscordSVG maxHeight="100%" maxWidth="100%" width="100%" />
+          </Box>
+        </a>
```
Comment (MarioBatalha) [inline components/layout/footer/index.tsx]: Use link from next instead anchor

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: components/layout/footer/index.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    </Box>
+    <SocialSection />
+    <Box
+      py="m"
+      width="100%"
+      display="flex"
+      borderTop="1px solid"
+      justifyContent="center"
+      borderColor="outlineVariant"
+    >
```
Comment (KipandaJr) [inline components/layout/footer/index.tsx:51]: check this because this dosent need be `100%`

Comment (KipandaJr) [inline components/layout/footer/index.tsx]: check this because this dosent need be `100%`

**Review by KipandaJr — CHANGES_REQUESTED** [review]

File: components/layout/footer/index.tsx (hunk 5 above)
Comment (atilasousa) [reply components/layout/footer/index.tsx]: please recheck this, because if i remove the 100%, the border won't be like in the figma

Comment (KipandaJr) [reply components/layout/footer/index.tsx]: i check try use the `box variant container` or follow how we do on `sui coins`

File: components/layout/footer/socials-section.tsx [hunk 6]
```diff
@@ -0,0 +1,78 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+
+import { DiscordSVG, GithubSVG, TelegramSVG, XSVG } from '@/components/svg';
+const SocialSection = () => (
+  <Box my="2xl">
+    <Box fontFamily="Proto">
+      <Typography
+        size={'medium'}
+        variant={'body'}
+        fontWeight="500"
```
Comment (MarioBatalha) [inline components/layout/footer/socials-section.tsx]: If you added the right size and variant you don't need fontWeight and fontFamily

File: components/svg/discord.tsx [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+      fill="#1B1B1F"
```
Comment (MarioBatalha) [inline components/svg/discord.tsx]: currentColor here

File: components/svg/heart.tsx [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+export const Heart: FC<SVGProps> = ({ maxWidth, maxHeight, ...props }) => (
+  <svg
+    style={{ maxWidth, maxHeight }}
+    viewBox="0 0 19 18"
+    fill="none"
+    {...props}
+  >
+    <path
+      d="M3.12891 2.66236C5.14439 0.940295 7.73973 1.25414 9.49466 2.7878C11.2476 1.25355 13.8153 0.961966 15.8532 2.65579C18.0834 4.50941 18.3512 7.67071 16.6339 9.88058C15.8515 10.8874 14.1662 12.5123 12.7247 13.8474C11.9943 14.5239 11.3107 15.1403 10.8099 15.5875C10.5594 15.8111 10.3544 15.9926 10.2119 16.1184C10.1406 16.1813 10.085 16.2302 10.047 16.2635L9.98844 16.3148L9.49495 16.746L9.00116 16.3146L9.00039 16.3139L8.94289 16.2635C8.90494 16.2302 8.84927 16.1813 8.778 16.1184C8.63547 15.9926 8.43048 15.8111 8.17999 15.5875C7.67919 15.1403 6.99563 14.5239 6.26523 13.8474C4.82372 12.5123 3.13845 10.8874 2.35603 9.88058C0.63207 7.66219 0.948817 4.52508 3.12891 2.66236Z"
+      fill="#E53E3E"
```
Comment (MarioBatalha) [inline components/svg/heart.tsx]: currentColor here too

File: components/svg/ipx.tsx [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M10.3804 0.666504C12.1955 0.666504 13.6667 2.14345 13.6667 3.96509V7.35803H10.2864C8.4715 7.35803 7.00004 5.88108 7.00004 4.05944V0.666504H10.3804ZM16.953 7.35803C15.1382 7.35803 13.6667 8.83498 13.6667 10.6566V14.0496H17.047C18.8619 14.0496 20.3334 12.5726 20.3334 10.751V7.35803H23.7137C25.5286 7.35803 27 5.88108 27 4.05944V0.666504H23.6197C21.8048 0.666504 20.3334 2.14345 20.3334 3.96509V7.35803H16.953ZM17.047 20.7411C18.8619 20.7411 20.3334 22.218 20.3334 24.0397V27.4326H16.953C15.1382 27.4326 13.6667 25.9557 13.6667 24.134V20.7411H17.047ZM7.00004 14.0496H3.6197C1.80484 14.0496 0.333374 12.5726 0.333374 10.751V7.35803H3.71371C5.52858 7.35803 7.00004 8.83498 7.00004 10.6566V14.0496ZM7.00004 14.0496V17.4425C7.00004 19.2641 8.4715 20.7411 10.2864 20.7411H13.6667V17.3481C13.6667 15.5265 12.1955 14.0496 10.3804 14.0496H7.00004ZM23.6197 14.0496C21.8048 14.0496 20.3334 15.5265 20.3334 17.3481V20.7411H23.7137C25.5286 20.7411 27 19.2641 27 17.4425V14.0496H30.3804C32.1952 14.0496 33.6667 12.5726 33.6667 10.751V7.35803H30.2864C28.4715 7.35803 27 8.83498 27 10.6566V14.0496H23.6197Z"
+      fill="#0053DB"
```
Comment (MarioBatalha) [inline components/svg/ipx.tsx]: currentColor here

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: components/layout/footer/index.tsx [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+  <Box
+    pr="s"
+    pb="xl"
+    pl="xs"
+    pt="8xl"
+    fontSize="s"
+    display="flex"
+    fontFamily="Proto"
+    alignItems="center"
+    flexDirection="column"
```
Comment (KipandaJr) [inline components/layout/footer/index.tsx]: u really need this props here? like a `fontFamily`?

File: components/layout/footer/index.tsx [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+    textTransform="uppercase"
+  >
+    <Box textAlign="center" mb="l" maxWidth="96%" minWidth="90rem">
+      <Typography
+        size="medium"
+        variant="body"
+        fontWeight="500"
+        fontFamily="Proto"
```
Comment (KipandaJr) [inline components/layout/footer/index.tsx]: dont need this

File: components/layout/footer/index.tsx [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+      >
+        This DApp is maintained &
+      </Typography>
+      <Typography
+        gap="0.25rem"
+        size="medium"
+        display="flex"
+        variant="body"
+        fontFamily="Proto"
```
Comment (KipandaJr) [inline components/layout/footer/index.tsx]: dont need this

File: components/layout/footer/socials-section.tsx [hunk 13]
```diff
@@ -0,0 +1,77 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+
+import { DiscordSVG, GithubSVG, TelegramSVG, XSVG } from '@/components/svg';
+const SocialSection = () => (
+  <Box my="2xl">
+    <Box fontFamily="Proto">
+      <Typography
+        size="medium"
+        variant="body"
+        fontFamily="Proto"
```
Comment (KipandaJr) [inline components/layout/footer/socials-section.tsx]: dont need this

**Review by KipandaJr — CHANGES_REQUESTED** [review]

File: components/layout/footer/index.tsx [hunk 14]
```diff
@@ -0,0 +1,47 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import { HeartSVG, IPXSVG } from '@/components/svg';
+
+import SocialSection from './socials-section';
+
+const Footer: FC = () => (
+  <Box
+    fontSize="s"
```
Comment (KipandaJr) [inline components/layout/footer/index.tsx]: `Box` need have a props to `Box` not to `Typography`

File: components/layout/footer/index.tsx [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+    display="flex"
+    flexDirection="column"
+    textTransform="uppercase"
+  >
+    <Box textAlign="center" mb="l">
+      <Typography size="large" variant="label" fontWeight="500">
```
Comment (KipandaJr) [inline components/layout/footer/index.tsx]: remove this `fontWeight`

File: components/svg/heart.tsx (hunk 8 above)
Comment (KipandaJr) [inline components/svg/heart.tsx]: Check this too

**Review by KipandaJr — CHANGES_REQUESTED** [review]

File: components/layout/footer/index.tsx [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+import SocialSection from './socials-section';
+
+const Footer: FC = () => (
+  <Box
+    fontSize="s"
+    display="flex"
+    flexDirection="column"
+    textTransform="uppercase"
+  >
+    <Box textAlign="center" mb="l">
```
Comment (MarioBatalha) [inline components/layout/footer/index.tsx]: This box is like a div and if you want to align elements inside them use display

File: components/layout/footer/index.tsx [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+      <Typography size="large" variant="label" fontWeight="500">
+        This DApp is maintained &
+      </Typography>
+      <Typography
```
Comment (MarioBatalha) [inline components/layout/footer/index.tsx:19]: I believe use text-align center is a better approach than use display cuz this element is like text tag (h and p)

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

**Review by KipandaJr — APPROVED** [review]

**Review by MarioBatalha — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**